.php_cs.dist

```diff
@@ -8,200 +8,198 @@ For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.
 EOF;
 
+$finder = PhpCsFixer\Finder::create()
+    ->files()
+    ->in(__DIR__ . '/src')
+    ->in(__DIR__ . '/tests/_files')
+    ->in(__DIR__ . '/tests/basic')
+    ->in(__DIR__ . '/tests/end-to-end')
+    ->in(__DIR__ . '/tests/fail')
+    ->in(__DIR__ . '/tests/unit')
+    ->notName('*.phpt')
+    ->notName('ClassWithAllPossibleReturnTypes.php');
+
 return PhpCsFixer\Config::create()
+    ->setFinder($finder)
     ->setRiskyAllowed(true)
-    ->setRules(
-        [
-            'align_multiline_comment' => true,
-            'array_indentation' => true,
-            'array_syntax' => ['syntax' => 'short'],
-            'binary_operator_spaces' => [
-                'operators' => [
-                    '=' => 'align',
-                    '=>' => 'align',
-                ],
-            ],
-            'blank_line_after_namespace' => true,
-            'blank_line_before_statement' => [
-                'statements' => [
-                    'break',
-                    'continue',
-                    'declare',
-                    'do',
-                    'for',
-                    'foreach',
-                    'if',
-                    'include',
-                    'include_once',
-                    'require',
-                    'require_once',
-                    'return',
-                    'switch',
-                    'throw',
-                    'try',
-                    'while',
-                    'yield',
-                ],
+    ->setRules([
```

Comment:
I have removed the wrapping of the argument list (which contains a single argument only) of `setRules()`, and instead wrap the argument itself only.